.changeset/fuzzy-shrimps-double.md

```diff
@@ -0,0 +1,6 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-studio': patch
+---
+
+Include query parameter when generating service test data. Support hashing test data for service tests.
```

Comment:
Let's split this changeset:

1. `@finos/legend-graph`: patch
```
Fix a problem with generating test data by properly including query parameter. Also, we now make use of the `test-data` generation's feature to `anonymize` sensitive data if needed.
```

2. `@finos/legend-studio`: patch
```
(on message)
```